examples/esp32/WebSocketClientSSLBundle/src/main.cpp

```diff
@@ -0,0 +1,91 @@
+#include "main.hpp"
+
```

Comment:
this file does not look like a example for the Client.
may the code of examples/esp32/WebSocketClientSSLBundle/example.cpp needs to be here?

Reply:
I updated the main.cpp to handle the websocket connection. The problem is that for the example to be useful it has to include the `x509_crt_bundle` file which I only know how to link with platformIO. I also added an INO version that does everything except include the bundle.

Reply:
I pushed changes to fix the Arduino build

Reply:
I updated the build with a workaround. I am not deep enough in this build system or the incbin library, but that should be the path forward.

Reply:
looks like that `arduino-cli` tries to build the ino and the main.cpp which results in conflicts.

Reply:
I separated the the PIO build into a separate folder. That might work. The build works locally now for me